design-patterns/gc-intro/samples/gc-intro-image.html

```diff
@@ -0,0 +1,11 @@
+<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
```

Comment:
```suggestion
<div class="bg-center bg-cover" data-bgimg-srcset="img/1x1.png 991w, img/ip-cover-image-1200x726.jpg 992w">
```
We are missing the asset img/1x1.png, but should we use that workaround or to update the bgimg-srcset to support a state where there is "no image" for the specified view port?

Reply:
Added [PR ](https://github.com/wet-boew/wet-boew/pull/9745) which the current PR is now dependent on.